composer.json

```diff
@@ -15,6 +23,8 @@
     "phpcs-dev": "\"vendor/bin/phpcs\"",
     "run-phpcs": [
       "@phpcs-dev"
-    ]
+    ],
+    "phpunit-dev": "\"vendor/bin/phpunit\"",
+    "phpunit-integration-dev": "\"vendor/bin/phpunit\" -c \"phpunit.integration.xml.dist\""
```

Comment:
Maybe add additional scripts to skip creating code coverage report (if it's generated in the first place).

Or just add to the readme that people can pass additional arguments in the composer script using 

```bash
composer phpunit-dev -- --no-coverage
```

And `--filter` for running single test suite.

Also adding `@php` will automatically resolve to whatever php process is currently being used.

https://getcomposer.org/doc/articles/scripts.md#executing-php-scripts

Reply:
Just took a look at the xml file and there is no `<logging>` part. Should this be added?